package.json

```diff
@@ -47,7 +47,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@sasjs/adapter": "3.9.4",
+    "@sasjs/adapter": "3.10.3",
```

Comment:
there is a newer version of `@sasjs\adapter@3.10.8`

Reply:
bumped